package.json

```diff
@@ -29,6 +29,7 @@
     "file-saver": "2.0.5",
     "immutability-helper": "3.1.1",
     "jquery": "3.6.1",
+    "js-cookie": "^3.0.1",
```

Comment:
No caret please. Oh right, and this got here by accident anyway, right?

Reply:
yes it's an accident, for some reason my a4linking doesn't work without it any more, need to troubleshoot that but for now have just been installing it while linked, sorry forgot to remove